### ISSUE TITLE
Update CentoOS/RHEL containerd version to 1.5 to avoid docker "hang" issues  

### ISSUE THREAD

**Issue by jakommo** [elastic/ansible-elastic-cloud-enterprise#167]
TL;DR: We should update the containerd version in RHEL/CentOS (i.e. [here](https://github.com/elastic/ansible-elastic-cloud-enterprise/blob/master/vars/os_RedHat_8.yml#L24)) to `1.5.*` to fix runc the issue described bwelow and also match our [own install docs](https://www.elastic.co/guide/en/cloud-enterprise/3.4/ece-configure-hosts-rhel-centos-cloud.html#ece_rhel_8_or_centos_8), that use `sudo yum install -y docker-ce-20.10* docker-ce-cli-20.10* containerd.io-1.5.*` today (ECE 3.4).

This is a follow up to https://github.com/elastic/ansible-elastic-cloud-enterprise/pull/125 and https://github.com/elastic/ansible-elastic-cloud-enterprise/issues/121.

We worked with a user yesterday that had issues with ECE (or better docker), where newly created containers would be stuck in "Created" state, similar [to the issue above](https://github.com/elastic/ansible-elastic-cloud-enterprise/issues/121).

 In https://github.com/elastic/ansible-elastic-cloud-enterprise/pull/125 we pinned the containerd version to `1.4.3` for RHEL/CentOS and back then this was `containerd.io-1.4.3-3.1.el8.x86_64` and using runc `1.0.0-rc92`.
Today if one installs `1.4.3` it translates to `containerd.io-1.4.3-3.2.el8.x86_64` and that ships with runc `1.0.0-rc93` and is affected by https://github.com/opencontainers/runc/pull/2871 .

**Comment by jakommo** [comment]
Related https://github.com/elastic/ansible-elastic-cloud-enterprise/issues/135

**Comment by gigerdo** [comment]
Should be fixed via https://github.com/elastic/ansible-elastic-cloud-enterprise/pull/188
